src/modules/75-cf/components/FlagOptionsMenuButton/__tests__/FlagOptionsMenuButton.test.tsx

```diff
@@ -331,7 +337,7 @@ describe('FlagOptionsButton', () => {
   describe('Archiving instead of Delete', () => {
     const isArchivingFfOn = true
```

Comment:
Not a blocker in the least, but since `FF` is short for `Feature Flag`, not Featureflag`, then both f's should be capitalised. #justsayin 😛 
```suggestion
    const isArchivingFFOn = true
```